### PR TITLE
Special grey door requirements: Pink Brinstar Hopper Room

### PR DESCRIPTION
**Pull request by fdtaillefer**

### CODE REVIEW

File: region/brinstar/pink.json [hunk 1]
```diff
@@ -898,7 +898,13 @@
...
           "locks": [
             {
-              "unlock": [ "Kill All Enemies" ]
+              "unlock": [
+                {"obstacle":{
+                  "id": "A",
+                  "requires": ["never"]
+                }}
+              ],
+              "note": "To avoid redundant requirements, obstacle must be destroying by going to 3 (possibly via 2) and back."
```
Comment (DIProgan) [inline region/brinstar/pink.json]: destroyed

**Comment by DIProgan** [comment]
The new node 3 was never defined

**Comment by fdtaillefer** [comment]
> 
> 
> The new node 3 was never defined

It wasn't "never" defined, but I did change my mind like 4 times and maybe I ended up not getting it back in. Sigh..

**Comment by DIProgan** [comment]
Aand good

**Pull request merged**